packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -108,23 +108,22 @@ function enableReviewWorkflow({ strapi }) {
       const { joinTable } = strapi.db.metadata.get(target).attributes[morphBy];
       const { idColumn, typeColumn } = joinTable.morphColumn;
 
-      const connection = strapi.db.getConnection();
-
       // Execute an SQL query to insert records into the join table mapping the specified content type with the first stage of the default workflow.
       // Only entities that do not have a record in the join table yet are selected.
-      const selectStatement = connection
+      const selectStatement = strapi.db
+        .getConnection()
         .select({
           [idColumn.name]: 'entity.id',
```

Comment:
is this ok? shouldn't be 
```suggestion
          [idColumn.name]: entity.id,
```

Reply:
I think it should be in quotes. entity is an SQL alias and not a JS variable

Reply:
oh my bad! completely missed it was an alias just below